File: Java/pythagoras.java
//This program calculates the hypotenuse of a right-angled triangle using the Pythagorean Theorem.

package Java_Samples;

import java.util.Scanner;

public class pythagoras {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter side a : ");
        int a = sc.nextInt();
        System.out.print("Enter side b: ");
        int b = sc.nextInt();

        double hypo = Math.sqrt(Math.pow(a, 2)+Math.pow(b, 2));

        System.out.println("Hypotenuse of the triangle is : "+hypo);
        sc.close();
    }
    
}

File: C/pythagoras.c
//This program calculates the hypotenuse of a right-angled triangle given the lengths of the other two sides.

#include<stdio.h>
#include<math.h>

int main(){
    int a,b;
    printf("Enter the length of side a and b : ");
    scanf("%d%d",&a,&b);

    double hypo = sqrt(pow(a,2)+pow(b,2));

    printf("Hypotenuse of Triangle : %.2lf",hypo);

    return 0;
}